server/pkg/accounts/accounts.go

```diff
@@ -3,14 +3,39 @@ package accounts
 import (
 	"fmt"
 	"os"
+	"errors"
 
 	"github.com/darwinfroese/hacksite/server/models"
 	"github.com/darwinfroese/hacksite/server/pkg/auth"
 	"github.com/darwinfroese/hacksite/server/pkg/database"
 )
 
+const (
+	// UnathorizedErrorMessage is used to determine if the error is because of unathorized account
+	UnathorizedErrorMessage = "Account is invalid"
+)
+
 // CreateAccount will create an account and insert it into the database
 func CreateAccount(db database.Database, account *models.Account) error {
+
+	//Check if the username already exists
+	_, invalidUsername := db.GetAccount(account.Username)
+	if invalidUsername != nil {
+		//Username is not in use
+	} else {
+		invalidUsername = errors.New("Username is taken")
```

Comment:
Can you make these exported constants, similar to the general unauthorized message. This way we can grab them and use them in logic in other packages like we do with the other unauthorized message.

Reply:
What exactly should be exported constants? The invalidUsername // invalidEmail variables? Or the "X is taken // Already in use" strings?

Reply:
The strings